src/worker/dump_db.rs

```diff
@@ -246,6 +253,22 @@ impl Drop for DumpTarball {
     }
 }
 
+struct InvalidateCaches {}
+
+impl InvalidateCaches {
+    pub fn create(client: &Client, target_name: &str) -> Result<(), PerformError> {
+        let cloudfront = CloudFront::from_environment()
```

Comment:
the `CloudFront` instance is already available on the `env: &Environment` of the parent job as `env.cloudfront`, so we don't need to recreate it for every job. I guess we could do something similar for Fastly too.

creating them upfront ensures that the background worker would fail on startup if the necessary env vars are not correctly set, instead of failing once the corresponding background job tries to run (which then pages the on-call team 🙈).